packages/babel/src/transformation/file/index.js

```diff
@@ -106,7 +107,9 @@ export default class File {
 
 
   /**
-   * [Please add a description.]
+   * Initialize this.opts (construct, normalize, set defaults).
```

Comment:
It technically doesn't touch `this.opts`, but other than that doesn't it have everything to do with initializing `this.opts`? Isn't that its sole purpose and don't all passed opts go through this method when constructing an instance? I thought it would be clearer to shortcut the description to reflect that, but if you prefer I can make it more generic and technically accurate. Either way I'll drop the "construct" part as that is an artefact of my mistake about the return value.